### PR TITLE
add fluid terminal

### PR DESCRIPTION
**Pull request by asdflj**
add fluid terminal
动画

### CODE REVIEW

**Review by Connor-Colenso — CHANGES_REQUESTED** [review]
Please make sure the terminal does not overlap with NEI, similar to what https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/143 implemented for a standard terminal.

File: src/main/java/com/glodblock/github/client/me/FluidRepo.java [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+        return this.dsp.get(idx);
+    }
+
+    void setSearch(final String search) {
+        this.searchString = search == null ? "" : search;
+    }
+
+    public void postUpdate(final IAEItemStack is) {
+        IAEFluidStack fluid = ItemFluidDrop.getAeFluidStack(is);
+        IAEItemStack i = AEItemStack.create(is.getItemStack().setStackDisplayName(fluid.getFluid().getLocalizedName()));
```
Comment (GlodBlock) [inline src/main/java/com/glodblock/github/client/me/FluidRepo.java:93]: you make a display only drop item like the fluid packet, so the name will look like normal and no strange tooltip

File: src/main/java/com/glodblock/github/client/gui/container/FCBaseFluidMonitorContain.java [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+import net.minecraftforge.fluids.IFluidContainerItem;
+import org.apache.commons.lang3.tuple.MutablePair;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.nio.BufferOverflowException;
+import java.util.HashMap;
+
+public class FCBaseFluidMonitorContain extends AEBaseContainer implements IConfigManagerHost, IConfigurableObject, IMEMonitorHandlerReceiver<IAEFluidStack> {
+    private final SlotRestrictedInput[] cellView = new SlotRestrictedInput[5];
```
Comment (GlodBlock) [inline src/main/java/com/glodblock/github/client/gui/container/FCBaseFluidMonitorContain.java:57]: i suggest to override the `transferStackInSlot` here so people can hold shift the put fluids into the terminal

**Review by GlodBlock — COMMENTED** [review]
also you would draw better a different texture, it is same with normal terminal now

File: src/main/java/com/glodblock/github/client/gui/container/FCBaseFluidMonitorContain.java (hunk 2 above)
Comment (asdflj) [reply src/main/java/com/glodblock/github/client/gui/container/FCBaseFluidMonitorContain.java:57]: 7N$05WJH$~~G$QE~FM7WH~A

Comment (asdflj) [reply src/main/java/com/glodblock/github/client/gui/container/FCBaseFluidMonitorContain.java:57]: i well do it

**Review by GlodBlock — APPROVED** [review]

**Pull request merged**